lib/pdn_management/Kconfig

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig PDN_MANAGEMENT
+	bool "PDN Management for nRF9160"
+	select BSD_LIBRARY
```

Comment:
Wouldn't it make more sense to make this library dependent on BSD_LIBRARY? I.e. `depends on BSD_LIBRARY`.

This way it will only get visible once we get bsdlib enabled.